pinot-core/src/test/java/org/apache/pinot/core/operator/filter/BitmapCollectionTest.java

```diff
@@ -0,0 +1,237 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.core.operator.filter;
+
+import org.roaringbitmap.buffer.ImmutableRoaringBitmap;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+
+public class BitmapCollectionTest {
+
+  @DataProvider
+  public static Object[][] andCardinalityTestCases() {
+    return new Object[][]{
```

Comment:
(code format) This does not follow the auto-formatter rule. You may wrap it with `//@formatter:off` and `//@formatter:on` to avoid future re-formatting on this block. Same for other manually formatted blocks

Reply:
I'll just reformat it

Reply:
done